bin/generate-page

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+require __DIR__ . '/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php';
+
+use CodeIgniter\CLI\CLI;
+
+$reflection = new ReflectionClass('Translations\Tests\AbstractTranslationTestCase');
+$locales    = $reflection->getStaticPropertyValue('locales') ?? [];
```

Comment:
`AbstractTranslationTestCase::$locales` is public. Why do you need to use reflection?

Reply:
Probably, because I'm blind 😅 
Thanks!